### DISCUSSION TITLE
prometheus-operator-0prometheusCustomResourceDefinition.yaml syntax errors

### DISCUSSION THREAD

**Discussion by lin-ket** [prometheus-operator/kube-prometheus#1474] [Q&A]
The CustomResourceDefinition "prometheuses.monitoring.coreos.com" is invalid: metadata.annotations: Too long: must have at most 262144 bytes
![image](https://user-images.githubusercontent.com/49556072/139818298-0d780ec7-7f5c-48b3-9372-a626923aabaa.png)

**Comment by paulfantom** [comment]
Use `kubectl create -f <file>`. Relevant: https://github.com/prometheus-operator/prometheus-operator/issues/4355

Could you create a PR to fix our docs?
